readme.md

```diff
@@ -61,14 +57,21 @@ to run as relay:
 
 Build requirements:
 
+* Git
 * CMake
 * C++ 17 capable C++ compiler
-* gcovr (if generating test coverage with gcc)
 * libuv >= 1.27.0
 * libsodium >= 1.0.18
+* libcurl >= 7.58.0
+* gcovr (if generating test coverage with gcc)
 
 ### Linux
 
+additional build requirements:
+
+* libcap >= 2.25 (Linux only)
```

Comment:
this is not a build dependency. setcap is only for source based installs and it's provided by `libcap2-bin` on debian.